README.md

```diff
@@ -25,6 +25,12 @@ $ ./cove/manage.py runserver
 
 CoVE is based on Django and be deployed using the deployment mechanisms as defined in the [Django docs](https://docs.djangoproject.com/en/3.1/howto/deployment/)
 
+### Run tests
+
+Set up [chromedriver](<https://chromedriver.chromium.org/getting-started>), then run:
+
+`DJANGO_SETTINGS_MODULE=cove.settings py.test cove/cove_360`
+
```

Comment:
This isn't the right path for the settings, there is no such file `cove.settings.py`. We don't have more than one settings module so omitting it is fine (it should be `DJANGO_SETTINGS_MODULE=cove_project.settings` if we really wanted to specify it)

Reply:
I did run the tests with this command. But I got a weird behavior. 

I tried the command that you said and it doesn't find `cove_project`. I don't know if `cove` needs to be specified before.

Reply:
Hmm maybe making sure to be in the `cove` directory makes the difference here (that's what the GH actions does) , the cove directory itself isn't a python module (has no __init__.py ) because the python module 'cove' name is unfortunately already used by libcoveweb.

Reply:
I can run the tests if I'm in `cove` but for browser tests, it is still asking me the chromedriver. I have checked how the autoinstaller it is set up and it all looks fine to me.